### PR TITLE
Cancel audit warning messages after group disable

### PR DESCRIPTION
**Pull request by rra**
If an audited group is disabled, cancel all the audit warning
messages in addition to marking the audit as complete.

When disabling a role user, both mark the audit as complete and
cancel all the audit warning messages.

Add a way to set an environment variable to disable headless mode
when running Selenium tests, since it helps with debugging new
tests.

### CODE REVIEW

File: grouper/fe/templates/service.html [hunk 1]
```diff
@@ -26,7 +26,8 @@
     <!-- Enable/Disable button. -->
```
Comment (wrodriguez-dbx) [inline grouper/fe/templates/service.html]: Nit: could we rename this file to `role-user.html` while we're here?

File: itests/fe/audits_test.py [hunk 2]
```diff
@@ -16,6 +18,65 @@
...
+        audit_modal = page.get_audit_modal()
+        audit_modal.click_close_button()
+        page.wait_until_audit_modal_clears()
+        page.click_disable_button()
+        modal = page.get_disable_modal()
+        modal.confirm()
+
+        assert page.subheading == "some-group (disabled)"
+
+    # And now all of the email messages should be marked sent except the non-async one.
```
Comment (wrodriguez-dbx) [inline itests/fe/audits_test.py]: Ah, non-async meaning immediate, which is still an `AsyncNotification`. Was confused for a sec :)

File: grouper/fe/templates/service.html (hunk 1 above)
Comment (rra) [reply grouper/fe/templates/service.html]: Good idea. Done.

File: itests/fe/audits_test.py (hunk 2 above)
Comment (rra) [reply itests/fe/audits_test.py]: Ah, yes, clarified the comment.

**Pull request merged**